src/Database.m

```diff
@@ -1776,6 +1779,8 @@ -(void)initFolderArray
                 NSInteger newItemId = [results stringForColumnIndex:0].integerValue;
                 NSInteger newParentId = [results stringForColumnIndex:1].integerValue;
                 NSString * name = [results stringForColumnIndex:2];
+				if (name == nil) // Paranoid check because of https://github.com/ViennaRSS/vienna-rss/issues/877
+					name = [Database untitledFeedFolderName];
```

Comment:
Can we please get `{ }` around this `if` statement? :P

Reply:
Agreed 😺. But did you see [this](https://github.com/barijaona/vienna-rss/commit/31c2b0dc6f7c7ac25f986d2a82945f950ff67cce#commitcomment-24067047) comment ? and [this one](https://github.com/lapcat/vienna-rss/pull/2#issuecomment-326807684) ?
@lapcat's # 4 point seems quite valid to me (even if I do prefer the more concise Spotify style).

My personal opinion is that we can remain flexible, at least until the beginning of the 3.3 development cycle. At this point, we should make a step forward by applying fine tuned uncrustify filtering to a subset of the code base.
Just my 2 cents. And I don't want to start a religious war between tab and spaces proponents 😆

Reply:
Note that `CONTRIBUTING.md` says (emphasis added) : 
> __try__ to follow the (...) style.